lib/mixins/semverist/directoryToSemverishObject.js

```diff
@@ -23,10 +22,17 @@ module.exports = superclass => class extends superclass {
    *   An string representing a path to a semverish directory.
    */
   setSemverishObject(semverishDirectoryPath) {
-    const root = semverishDirectoryPath;
+    const root = path.join(
+      this.getSemveristConfig().callPath,
+      semverishDirectoryPath
+    );
     const semverishObject = {};
+    const pathology = path.join(
+      this.getSemveristConfig().callPath,
+      semverishDirectoryPath
+    );
     // Walk this directory for directories.
-    const directories = klawSync(this.getResolvedPath(semverishDirectoryPath), {nofile: true});
+    const directories = klawSync(pathology, {nofile: true});
```

Comment:
What's happening here and does this happen during runtime?

Reply:
yes it is run time. if you are building up your semverist object via directory. This is one of the sync i/o things i will be addressing in subsequent efforts. You always have the option of passing in an object if you want to do the heavy lifting yourself though.